adoc/retail_migr_chap_prepare.adoc

```diff
@@ -0,0 +1,26 @@
+[[retail_migr.chap.prepare]]
+= Prepare to Migrate
+
+include::entities.adoc[]
+
+
+[[retail.sect.prepare]]
+== Before you Begin
+
+Before you begin migration, there are a few things you need to consider:
+
+* Something something
+* (environment, planning, etc)
+* ...
```

Comment:
Can you add some clearly visible mark that this is something TBD? I fear once this PR is merged we may miss this still needs to be filled in.